squad/core/notification.py

```diff
@@ -75,7 +75,9 @@ def recipients(self):
                 if not self.previous_build or \
                    len(self.comparison.regressions) == 0:
                     continue
-
+            elif subscription.notification_strategy == Subscription.NOTIFY_ON_ERROR:
+                if len(self.build.test_jobs.filter(job_status="Incomplete")) == 0:
```

Comment:
This is LAVA specific. I propose to use a setting from project_settings to decide on 'error' situation.